src/constants.rs

```diff
@@ -5,6 +5,7 @@ pub(crate) const BN_LIMB_WIDTH: usize = 64;
 pub(crate) const BN_N_LIMBS: usize = 4;
 pub(crate) const NUM_FE_WITHOUT_IO_FOR_CRHF: usize = 17;
 pub(crate) const NUM_FE_FOR_RO: usize = 9;
+pub(crate) const NUM_IO_IN_NOVA: usize = 2;
```

Comment:
We're using this constant in the Supernova module: this is going to confuse the heck out of people. Would something more specific help? maybe `NIO_NOVA_FOLD`?